packages/fast-components-react-msft/src/call-to-action/call-to-action.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from "react";
+import { Foundation, HandledProps } from "@microsoft/fast-components-react-base";
+import Button from "../button/index";
+import { ButtonAppearance, IMSFTButtonClassNameContract } from "../button/button";
+import { CallToActionAppearance, ICallToActionHandledProps } from "./call-to-action.props";
+import { ICallToActionClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts-msft";
+import { get } from "lodash-es";
+import { glyphArrowright } from "@microsoft/fast-glyphs-msft";
+import { applyLocalizedProperty, localizeSpacing, toPx } from "@microsoft/fast-jss-utilities";
+import { IDesignSystem, withDesignSystemDefaults } from "@microsoft/fast-components-styles-msft";
+import { ComponentStyles } from "@microsoft/fast-jss-manager";
+import { Direction } from "@microsoft/fast-application-utilities";
+
+// Since MSFT button is already styled, we need to override in this way to alter button classes
+const styles: ComponentStyles<Partial<IMSFTButtonClassNameContract>, IDesignSystem> = {
```

Comment:
can we move this to the `fast-components-styles-msft` package as a secondary export named `calltToActionButtonOverrides`

Reply:
missed this; good catch @janechu